docs/docs/working-with-Etsy-in-Gatsby.md

```diff
@@ -0,0 +1,68 @@
+---
+Title: Working with Etsy in Gatsby
```

Comment:
```suggestion
title: Working with Etsy in Gatsby
```

The function that generates these pages is looking for "title". This change gets the title to show up at the top of the guide.

Reply:
Good find, @AishaBlake. That’s another case where it should be discovered by running the docs site locally; same  with the other formatting issues.